source/nodejs/adaptivecards-site/package.json

```diff
@@ -4,12 +4,13 @@
 	"private": true,
 	"scripts": {
 		"clean": "hexo clean",
-		"build": "hexo generate",
```

Comment:
minor: can you make it default to window maximized? It's a little weird me that I clicked preview, and I didn't see it

![image](https://user-images.githubusercontent.com/6290692/171751391-baa2a7cf-52e3-4012-a1d9-711cf32ca598.png)

Reply:
@licanhua @aarongustafson Do you have any ideas for this? I tried using `viewport-fit=cover` in the html and `"display": "fullscreen"` in the manifest, but neither made a change.

I think the fix would be relatively straight forward in JS, but we don't want the screen to visibly resize when the app loads.

Reply:
I opened an issue for this: https://github.com/microsoft/AdaptiveCards/issues/7498